dao/src/main/java/uk/gov/moj/sdt/dao/BulkSubmissionDao.java

```diff
@@ -94,24 +64,30 @@ public IBulkSubmission getBulkSubmission(final IBulkCustomer bulkCustomer, final
             criteriaBuilder.lower(root.get("customerReference")),
             customerReference.toLowerCase()
         );
+        Predicate sdtDatePredicate = createDatePredicate(criteriaBuilder, root, dataRetention);
+
         TypedQuery<BulkSubmission> typedQuery = getEntityManager().createQuery(criteriaQuery.select(root)
                                                                                    .where(
-                                                                                       sdtCustomerPredicate,
-                                                                                       customerReferencePredicate,
-                                                                                       createDatePredicate(criteriaBuilder, root, dataRetention)
+                                                                                       sdtCustomerPredicate
+                                                                                       ,customerReferencePredicate
+                                                                                       ,sdtDatePredicate
                                                                                    ));
 
+        LOGGER.debug("typedQuery.getResultList().size() == {}", typedQuery.getResultList().size());
```

Comment:
Is this debug line one that was used during development?  If so it may be able to be removed.  If it is removed then remove the similar line in the getBulkSubmissionBySdtRef() method.